core-web/libs/block-editor/src/lib/shared/services/suggestions/suggestions.service.ts

```diff
@@ -37,11 +37,14 @@ export class SuggestionsService {
     getContentlets({
         contentType,
         filter,
-        currentLanguage
+        currentLanguage,
+        contentletIdentifier
     }: ContentletFilters): Observable<DotCMSContentlet[]> {
+        const identifierQuery = contentletIdentifier ? `-identifier:${contentletIdentifier}` : '';
```

Comment:
No need for this conditional. We should expect always to have the `contentletIdentifier`.

Reply:
@rjvelazco `contentletIdentifier` is empty the first time a new Block Editor is open since it hasn't been saved.

Reply:
@manuelrojas That's true. My bad, good catch!